### PR TITLE
ZCS-1174: Universal UI- New Contact Group/Edit Contact Group

### PR DESCRIPTION
**Pull request by kalpeshhpatel**
Changes:

* Contacts.template:
  * Updated template for edit contact group
* Added/updated required CSS for `zm.css`, `dwt.css`, `common.css`, `skin.css`
* Added/updated required  properties in `skin.properties`
* ZmGroupView.js:
  * Change the style and icons for required operations
* ZmMsg.properties:
  * Change property strings as per requirement

https://jira.corp.synacor.com/browse/ZCS-1174

### CODE REVIEW

File: WebRoot/css/zm.css [hunk 1]
```diff
@@ -6616,3 +6604,159 @@ a.ZmLinkDisabled {
...
+	@GroupEditView@
+}
+
+.ZmContactView .ZmGroupViewWrapper {
+	@GroupEditViewWrapper@
+}
+
+.ZmContactView .ZmGroupViewWrapper input[type='text'],
+.ZmContactView .ZmGroupViewWrapper textarea {
+	@GroupEditViewFontSize@
```
Comment (tarangkhandelwal) [inline WebRoot/css/zm.css]: better to use something like GroupEditViewInput or sort of this?

File: WebRoot/css/zm.css [hunk 2]
```diff
@@ -6616,3 +6604,159 @@ a.ZmLinkDisabled {
...
+	@GroupAddMemberBtnWrapper@
+}
+
+.ZmGroupAddMemberBtn .ZButton {
+	@GroupAddMemberBtn@
+}
+
+.ZmGroupAddMemberBtn .ZButton:hover,
+.ZmGroupAddMemberBtn .ZButton:focus,
+.ZmGroupAddMemberBtn .ZButton:active,
```
Comment (tarangkhandelwal) [inline WebRoot/css/zm.css]: do we need to use both selectors..the pseudo selectors and equivalent class selectors . 
:hover & ZHover for example

Comment (kalpeshhpatel) [inline WebRoot/css/zm.css]: Without pseudo selectors should work, no harm in removing that.

File: WebRoot/css/zm.css (hunk 1 above)
Comment (kalpeshhpatel) [inline WebRoot/css/zm.css]: Done: https://github.com/Zimbra/zm-web-client/pull/46/files#diff-33f9c57f22cd743824b2918132c5d73bR6619

File: WebRoot/css/zm.css (hunk 2 above)
Comment (kalpeshhpatel) [inline WebRoot/css/zm.css]: Removed: https://github.com/Zimbra/zm-web-client/pull/46/files#diff-33f9c57f22cd743824b2918132c5d73bR6695

File: WebRoot/js/zimbraMail/abook/view/ZmGroupView.js [hunk 3]
```diff
@@ -41,7 +41,7 @@
 ZmGroupView = function(parent, controller) {
 	if (arguments.length == 0) return;
 	DwtComposite.call(this, {parent:parent, className:"ZmContactView", posStyle:DwtControl.ABSOLUTE_STYLE});
-	this.setScrollStyle(Dwt.CLIP); //default is clip, for regular group it's fine. (otherwise there's always a scroll for no reason, not sure why). For DL we change in "set"
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/abook/view/ZmGroupView.js]: comment doesn't match with default value

Comment (kalpeshhpatel) [inline WebRoot/js/zimbraMail/abook/view/ZmGroupView.js]: Fixed: https://github.com/Zimbra/zm-web-client/pull/46/files#diff-04a1d0a2f989ff409128b069b9946915L44

**Pull request merged**